src/Microsoft.TestPlatform.Common/Interfaces/Engine/TesthostProtocol/IExecutionManager.cs

```diff
@@ -24,21 +24,27 @@ public interface IExecutionManager
         /// Starts the test run with sources.
         /// </summary>
         /// <param name="adapterSourceMap"> The adapter Source Map.  </param>
+        /// <param name="package"> The packages which actually contain sources. A testhost can at max execute for one pakage at time
+        /// Package can be null if test source, & package are same
```

Comment:
nit: `&` doesn't seem to play well with xml doc comments.

Reply:
resolved